src/babel-register/index.js

```diff
@@ -1,3 +1,13 @@
+// unless we are running a prebuilt/distributable version of
+// kibana, automatically transpile typescript to js before babel
+if (!global.__BUILT_WITH_BABEL__) {
+  const { resolve } = require('path');
+  require('ts-node').register({
```

Comment:
Yeah, I thought that at first but then realized that once babel 7 is available it will really be babel-register again, so maybe it's fine that in the meantime it's a little more that just babel-register.